css/style.css

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please, consider removing a lot of unnecessary white spaces in your CSS. Leaving such white spaces leads to linter errors.
Kindly set up Github's Continuous Integration tool to help you catch such errors.